src/components/address-input/_macro.spec.js

```diff
@@ -87,126 +45,209 @@ describe('macro: address-input', () => {
-            const faker = templateFaker();
-            const inputSpy = faker.spy('input', { suppressOutput: true });
-
-            faker.renderComponent('address-input', {
-                ...EXAMPLE_AUTOSUGGEST_ADDRESS_MINIMAL,
-                ...EXAMPLE_MANUAL_INPUT_FIELDS,
-                isEditable: true,
+                value: '[params.value]',
+                attributes: '[params.attributes]',
+                error: '[params.error]',
+                name: '[params.name]',
+                mutuallyExclusive: '[params.mutuallyExclusive]',
+                APIDomain: '[params.APIDomain]',
+                APIDomainBearerToken: '[params.APIDomainBearerToken]',
+                APIManualQueryParams: '[params.APIManualQueryParams]',
+                allowMultiple: '[params.allowMultiple]',
+                mandatory: '[params.mandatory]',
+                instructions: '[params.instructions]',
+                autocomplete: '[params.autocomplete]',
+                isEditable: '[params.isEditable]',
+                ariaYouHaveSelected: '[params.ariaYouHaveSelected]',
+                ariaMinChars: '[params.ariaMinChars]',
+                minChars: '[params.minChars]',
+                ariaResultsLabel: '[params.ariaResultsLabel]',
+                ariaOneResult: '[params.ariaOneResult]',
+                ariaNResults: '[params.ariaNResults]',
+                ariaLimitedResults: '[params.ariaLimitedResults]',
+                ariaGroupedResults: '[params.ariaGroupedResults]',
+                groupCount: '[params.groupCount]',
+                moreResults: '[params.moreResults]',
+                tooManyResults: '[params.tooManyResults]',
+                resultsTitle: '[params.resultsTitle]',
+                resultsTitleId: '[params.resultsTitleId]',
+                noResults: '[params.noResults]',
+                typeMore: '[params.typeMore]',
+                errorTitle: '[params.errorTitle]',
+                errorMessageEnter: '[params.errorMessageEnter]',
+                errorMessageSelect: '[params.errorMessageSelect]',
+                errorMessageAPI: '[params.errorMessageAPI]',
+                errorMessageAPILinkText: '[params.errorMessageAPILinkText]',
+                options: '[params.options]',
+                manualLink: '[params.manualLink]',
+                manualLinkText: '[params.manualLinkText]',
             });
 
-            expect(inputSpy.occurrences).toContainEqual({
-                id: 'address-input-example-id-line2',
-                name: 'address-input-example-id-line2',
-                classes: 'ons-js-address-line2',
-                label: {
-                    text: 'Address line 2',
-                },
-                value: '12345 The Road',
-                width: '20@m',
-                error: { text: 'Server error: address line 2' },
+            test('THEN: it renders autosuggest component with expected parameters', () => {
```

Comment:
Maybe this should be "THEN: the provided attributes are passed through to the autosuggest component" because that is what it is testing